### DISCUSSION TITLE
Some extra Firefox Contain integration

### DISCUSSION THREAD

**Discussion by ustamills** [piroor/treestyletab#2879] [Ideas]
I use TST and Firefox Containers extensively. They manage my online life for work and personal. In order to corral all those tabs I also use the 'Simple Tab Rename' extension (STR) so I can have a top level tab for each container. This top-level tab contains all associated container tabs due to the skills of TST.

![image](https://user-images.githubusercontent.com/143219/116443011-269ad300-a821-11eb-8634-563cde20b248.png)

Those top-level tabs are just google.com tabs renamed to something helpful using STR.

**MY IDEA**

It would be incredibly helpful if empty top-level tabs, assigned to containers, could be a configurable startup thing. It would make starting up my work sessions MUCH easier. I couldn't find a way to make that happen just using TST.

My current solution works. But I have to create several empty tabs using TST, visit some sites, rename top level tabs (using STR).

**Comment by piroor** [comment]
[Container Bookmarks](https://addons.mozilla.org/firefox/addon/container-bookmarks/) provides ability to open tabs with specific container. It looks to help you to prepare initial container  tabs from bookmarks.
